local_postgres_docker_setup.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
```

Comment:
Actually, on second thought, and given my conversation with @kurtwheeler about the possible danger of running production databases on containers (without mounting them first to a volume on the local computer), I think I will create a whole separate issue for this, to decide how it would be implemented - I think it might be outside the scope of this PR.